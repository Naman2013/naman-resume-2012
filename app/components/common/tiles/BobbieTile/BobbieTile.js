import React from 'react';
import PropTypes from 'prop-types';
import { horizontalArrow } from 'styles/variables/iconURLs';
import style from './BobbieTile.style';

/**
  @BobbieTile
  This tile is hard coded to an expected HTML blob provided
  by the `getGuidePanels` response.
  40|60
  60|40
  50|50
  image and video placement support
  https://docs.google.com/document/d/1PK8X1clsV618gHQWtBnwRcFCVsTbpMhSbIYvnz8MW6Y/edit#
*/

const BobbieTile = ({
  title,
  readDuration,
  authorName,
  HTMLBlob,
}) => (
  <div className="root">
    <div className="tile-content-container">
      <h3 className="title">{title}</h3>
      <div className="post-meta-data">
        <ul>
          <li className="read-duration">{readDuration}</li>
          <li className="author-name">By {authorName} <img alt="" src={horizontalArrow} /></li>
        </ul>
      </div>
      <div
        className="html-blob-content-container"
        dangerouslySetInnerHTML={{ __html: HTMLBlob }}
      />
    </div>
    <style jsx>{style}</style>
  </div>
);

BobbieTile.propTypes = {
  title: PropTypes.string.isRequired,
  readDuration: PropTypes.string,
  authorName: PropTypes.string,
  HTMLBlob: PropTypes.string.isRequired,
};

BobbieTile.defaultProps = {
  readDuration: '',
  authorName: '',
};

export default BobbieTile;
