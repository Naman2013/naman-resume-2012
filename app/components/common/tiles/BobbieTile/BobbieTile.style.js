import css from 'styled-jsx/css';
import { primaryFont, secondaryFont } from 'styles/variables/fonts';
import { faintShadow } from 'styles/variables/shadows';
import { astronaut, geyser } from 'styles/variables/colors_tiles_v4';
import { resetMarginPadding } from 'styles/variables/utils';
import { screenMedium, screenLarge } from 'styles/variables/breakpoints';

export default css`
  div, h3, ul, li {
    ${resetMarginPadding}
  }

  .root {
    ${faintShadow}
    background: white;
  }

  .tile-content-container {
    padding: 40px;
  }

  .title {
    font-family: ${secondaryFont};
    font-size: 20px;
    color: ${astronaut};
    border-bottom: 1px solid ${geyser};
    font-weight: 300;
    padding-bottom: 20px;
  }

  .subtitle {
    list-style-type: none;
    font-family: ${primaryFont};
    color: ${astronaut};
    text-transform: uppercase;
    font-size: 10px;
    font-weight: bold;
    letter-spacing: 2px;
    margin: 20px 0;
    padding: 10px 0;
    border-bottom: 1px solid ${geyser};
    border-top: 1px solid ${geyser};
  }

  ul {
    list-style-type: none;
    font-family: ${primaryFont};
    color: ${astronaut};
    text-transform: uppercase;
    font-size: 10px;
    font-weight: bold;
    padding-top: 20px;
  }

  .author-name {
    display: flex;
    justify-content: space-between;
    align-items: center;
  }

  .__html-blob-content-container__,
  .read-duration {
    /* display: none; */
  }

  .__html-blob-content-container__ :global(ol) {
    margin-left: 40px;
    margin-bottom: 20px;
  }

  .__html-blob-content-container__ :global(ul) {
    margin-left: 40px;
    margin-bottom: 20px;
  }

  .__html-blob-content-container__ :global(li) {
    margin-bottom: 10px;
  }

  .__html-blob-content-container__ :global(p) {
    margin-top: 20px;
    margin-bottom: 20px;
  }

  .__html-blob-content-container__ :global(h1) {
    margin-top: 40px;
    margin-bottom: 10px;
  }

  .__html-blob-content-container__ :global(h2) {
    margin-top: 40px;
    margin-bottom: 10px;
  }

  .__html-blob-content-container__ :global(h3) {
    margin-top: 40px;
    margin-bottom: 10px;
  }

  .__html-blob-content-container__ :global(h4) {
    margin-top: 40px;
    margin-bottom: 10px;
  }

  .__html-blob-content-container__ :global(h5) {
    margin-top: 40px;
    margin-bottom: 10px;
  }

  @media ${screenMedium} {
    .__html-blob-content-container__,
    .read-duration {
      display: block;
    }

    .author-name img,
    .author-name span { display: none; }

    .tile-content-container {
      padding: 40px 50px;
    }

    ul {
      display: flex;
      padding: 10px 0;
      margin-bottom: 20px;
      border-bottom: 1px solid ${geyser};
    }

    .read-duration {
      border-right: 1px solid ${geyser};
      padding-right: 10px;
      margin-right: 10px;
    }
  }

  @media ${screenLarge} {
    .tile-content-container {
      padding: 80px;
    }

    .title {
      font-size: 24px;
    }
  }
`;
