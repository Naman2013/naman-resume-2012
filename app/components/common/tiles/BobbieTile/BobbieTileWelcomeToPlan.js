import React from 'react';
import PropTypes from 'prop-types';
import GuideExcerptTile from '../../../common/tiles/guide-excerpt-tile';
import styles from './BobbieTileWelcomeToPlan.styles';
import CMSStyle from './CMS.style';

/**
  @BobbieTileWelcomeToPlan
  This tile is hard coded to an expected HTML blob provided
  by the `getGuidePanels` response.
  40|60
  60|40
  50|50
  image and video placement support
  https://docs.google.com/document/d/1PK8X1clsV618gHQWtBnwRcFCVsTbpMhSbIYvnz8MW6Y/edit#
*/

const BobbieTileWelcomeToPlan = ({
  title,
  subTitle,
  planName,
  HTMLBlob,
  relatedGuide,
  hasRelatedGuide,
}) => (
  <div className="root">
    <div style={{ marginTop: '0px', marginBottom: '0px', padding: '0px' }} className="tile-content-container">
      <h1 style={{ marginBottom: '20px' }}>{title}</h1>
      <table className="middle-content" style={{ width: '100%' }}>
        <tr>
          <td style={{ width: hasRelatedGuide ? '60%' : '100%' }}>
            <div className="text-content">
              <h2>{subTitle}</h2>
              <div className="plan-name">{planName}</div>
              <div
                className="__html-blob-content-container__"
                dangerouslySetInnerHTML={{ __html: HTMLBlob }}
              />
            </div>
          </td>

          {hasRelatedGuide && (
            <td className="handle-hover" style={{ width: '40%' }}>
              <div className="guide-tile">
                <div className="guide-tile-frame">
                  <img alt="" src="https://vega.slooh.com/assets/v4/common/guide_corner.svg" className="top-left" />
                  <img alt="" src="https://vega.slooh.com/assets/v4/common/guide_corner.svg" className="top-right" />
                </div>
                <div className="guide-tile-frame bottom">
                  <img alt="" src="https://vega.slooh.com/assets/v4/common/guide_corner.svg" className="top-left" />
                  <img alt="" src="https://vega.slooh.com/assets/v4/common/guide_corner.svg" className="top-right" />
                </div>
                <h4 className="title">{relatedGuide.title}</h4>
                <h5 className="subTitle">{relatedGuide.subTitle}</h5>
              </div>
              <div className="excerpt-tile">
                <GuideExcerptTile
                  {...relatedGuide}
                />
              </div>
            </td>
          )}
        </tr>
      </table>
    </div>
    <style jsx>{styles}</style>
    <style jsx>{CMSStyle}</style>
  </div>
);

const {
  string,
  bool,
  shape,
} = PropTypes;

BobbieTileWelcomeToPlan.propTypes = {
  title: string.isRequired,
  subTitle: string.isRequired,
  planName: string,
  HTMLBlob: string.isRequired,
  relatedGuide: shape({}).isRequired,
  hasRelatedGuide: bool.isRequired,
};

BobbieTileWelcomeToPlan.defaultProps = {
  planName: '',
};

export default BobbieTileWelcomeToPlan;
