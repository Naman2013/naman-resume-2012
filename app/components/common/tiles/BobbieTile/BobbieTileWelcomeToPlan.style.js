import css from 'styled-jsx/css';
import { primaryFont, secondaryFont } from 'styles/variables/fonts';
import { faintShadow } from 'styles/variables/shadows';
import { astronaut, geyser, blue_tile_feat, golden } from 'styles/variables/colors_tiles_v4';
import { resetMarginPadding } from 'styles/variables/utils';
import { screenMedium, screenLarge } from 'styles/variables/breakpoints';


export default css`
  div, h3, ul, li {
    ${resetMarginPadding}
  }

  .root {
    background: white;
  }

  .tile-content-container {
    font-family:  ${secondaryFont};
  }

  .tile-content-container h1 {
    font-size: 20px;
    letter-spacing: 1px;
    padding: 25px 0;
    color: ${astronaut};
    border-bottom: 1px solid ${geyser};
    font-weight: 300;
  }

  .middle-content {
    width: 100%;
    display: flex;
    flex-flow: row wrap;
    align-content: space-between;
    margin-top: 40px;
  }

  .text-content {
    flex: 50%;
  }

  .text-content h2 {
     font-size: 18px;
     color: ${astronaut};
     letter-spacing: 1px;
     margin: 0;
     padding: 0;
  }

  .middle-content .guide-tile {
    position: relative;
    width: 100%;
    max-width: 400px;
    height: 250px;
    margin: 0 auto;      
    background: url(${blue_tile_feat});
    text-align: center;
  }

  .middle-content .guide-tile .title {
    margin: 0;
    margin-bottom: 10px;
    padding-top: 100px;
    font-family: ${primaryFont};
    color: ${golden};
    text-transform: uppercase;
    font-weight: 400;
    letter-spacing: 2px;
    font-size: 10px;
  }

  .middle-content .guide-tile .subTitle {
    margin: 0;
    font-family: ${secondaryFont};
    color: white;
    font-weight: 300;
    font-size: 20px;
    letter-spacing: 1px;
  }

  .guide-tile-frame {
    position: absolute;
    width: 100%;
    left: 50%;
    transform: translateX(-50%);
    display: flex;
    justify-content: space-between;
  }

  .bottom {
    transform: rotate(180deg);
    bottom: 0;
    left:0;
  }

  .top-right { transform: rotate(90deg) }

  .plan-name {
    list-style-type: none;
    font-family: ${primaryFont};
    color: ${astronaut};
    text-transform: uppercase;
    font-size: 10px;
    font-weight: bold;
    letter-spacing: 2px;
    margin: 20px 0;
    padding: 10px 0;
    border-bottom: 1px solid ${geyser};
    border-top: 1px solid ${geyser};
  }

  .author-name {
    display: flex;
    justify-content: space-between;
    align-items: center;
  }

  .__html-blob-content-container__ {
    font-size: 18px;
  }

  .__html-blob-content-container__,
  .read-duration {
    display: none;
  }

  .__html-blob-content-container__ :global(p) {
    margin-bottom: 10px;
  }

  @media ${screenMedium} {
    .__html-blob-content-container__,
    .read-duration {
      display: block;
    }

    .middle-content .guide-tile {
      width: 250px;
      margin: 0;
    }

    .text-content {
      padding-right: 40px;
    }
    
    .author-name img,
    .author-name span { display: none; }

    .tile-content-container {
      /* padding: 40px 50px; */
    }

    ul {
      display: flex;
      padding: 10px 0;
      margin-bottom: 20px;
      border-bottom: 1px solid ${geyser};
    }
  }

  @media ${screenLarge} {
    .tile-content-container {
      /* padding: 80px; */
    }

    .title {
      font-size: 24px;
    }
  }
`;
