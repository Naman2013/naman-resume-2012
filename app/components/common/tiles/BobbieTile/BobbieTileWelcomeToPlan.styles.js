import css from 'styled-jsx/css';
import { primaryFont, secondaryFont } from '../../../../styles/variables/fonts';
import {
  astronaut,
  geyser,
  blue_tile_feat,
  golden,
} from '../../../../styles/variables/colors_tiles_v4';
import { resetMarginPadding } from '../../../../styles/variables/utils';
import {
  screenMedium,
  screenLarge,
} from '../../../../styles/variables/breakpoints';

export default css`
  div,
  h3,
  ul,
  li {
    ${resetMarginPadding}
  }

  .root {
    background: white;
  }

  .tile-content-container {
    font-family: ${secondaryFont};
  }

  .tile-content-container h1 {
    font-size: 20px;
    letter-spacing: 1px;
    padding: 25px 0;
    color: ${astronaut};
    border-bottom: 1px solid ${geyser};
    font-weight: 300;
  }

  .middle-content {
    width: 50%;
    display: flex;
    flex-flow: row wrap;
    align-content: space-between;
    margin-top: 40px;
  }

  .text-content {
    flex: 50%;
  }

  .text-content h2 {
    font-size: 18px;
    color: ${astronaut};
    letter-spacing: 1px;
    margin: 0;
    padding: 0;
  }

  .middle-content .guide-tile {
    position: relative;
    width: 100%;
    max-width: 400px;
    height: 250px;
    margin: 0 auto;
    background: url(${blue_tile_feat});
    text-align: center;
  }

  .middle-content .guide-tile .title {
    margin: 0;
    margin-bottom: 10px;
    padding-top: 100px;
    font-family: ${primaryFont};
    color: ${golden};
    text-transform: uppercase;
    font-weight: 400;
    letter-spacing: 2px;
    font-size: 10px;
  }

  .middle-content .guide-tile .subTitle {
    margin: 0;
    font-family: ${secondaryFont};
    color: white;
    font-weight: 300;
    font-size: 20px;
    letter-spacing: 1px;
  }

  .guide-tile-frame {
    position: absolute;
    width: 100%;
    left: 50%;
    transform: translateX(-50%);
    display: flex;
    justify-content: space-between;
  }

  .bottom {
    transform: rotate(180deg);
    bottom: 0;
    left: 0;
  }

  .top-right {
    transform: rotate(90deg);
  }

  .plan-name {
    list-style-type: none;
    font-family: ${primaryFont};
    color: ${astronaut};
    text-transform: uppercase;
    font-size: 10px;
    font-weight: bold;
    letter-spacing: 2px;
    margin: 20px 0;
    padding: 10px 0;
    border-bottom: 1px solid ${geyser};
    border-top: 1px solid ${geyser};
  }

  .author-name {
    display: flex;
    justify-content: space-between;
    align-items: center;
  }

  .excerpt-tile {
    display: none;
  }

  .excerpt-tile :global(.guide-tile-root) {
    width: 250px;
    height: 250px;
  }

  .excerpt-tile :global(.guide-tile-root .actions) {
    width: auto;
    margin: unset;
  }

  @media ${screenMedium} {
    .middle-content .guide-tile {
      width: 250px;
      margin: 0;
    }

    .text-content {
      padding-right: 40px;
    }

    .author-name img,
    .author-name span {
      display: none;
    }

    .tile-content-container {
      /* padding: 40px 50px; */
    }

    ul {
      display: flex;
      padding: 10px 0;
      margin-bottom: 20px;
      border-bottom: 1px solid ${geyser};
    }

    .handle-hover:hover .excerpt-tile {
      display: block;
    }

    .handle-hover .excerpt-tile :global(.description) {
      white-space: nowrap;
      text-overflow: ellipsis;
      overflow: hidden;
    }

    .handle-hover .excerpt-tile :global(.title) {
      font-size: 18px;
    }

    .handle-hover .excerpt-tile :global(.sub-title) {
      padding: 12px 0;
    }

    .handle-hover:hover .guide-tile {
      display: none;
    }
  }

  @media ${screenLarge} {
    .tile-content-container {
      /* padding: 80px; */
    }

    .title {
      font-size: 24px;
    }
  }
`;
