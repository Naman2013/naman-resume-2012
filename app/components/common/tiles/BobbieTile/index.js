import BobbieTile from './BobbieTitle';

export default BobbieTile;
