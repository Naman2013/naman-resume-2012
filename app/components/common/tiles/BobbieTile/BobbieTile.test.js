import React from 'react';
import { shallow } from 'enzyme';
import BobbieTile from './BobbieTile';

describe('BobbieTile', () => {
  const shallowWrapper = shallow(<BobbieTile
    title="Check out this tutorial"
    HTMLBlob="<h1>Hello world...</h1>"
    readDuration="30"
    authorName="Paul Cox"
  />);
  it('should render correctly', () => {
    expect(shallowWrapper).toMatchSnapshot();
  });
});
