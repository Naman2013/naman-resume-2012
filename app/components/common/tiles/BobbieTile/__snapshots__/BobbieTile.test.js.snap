// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BobbieTile should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <BobbieTile
    HTMLBlob="<h1>Hello world...</h1>"
    authorName="Paul Cox"
    readDuration="30"
    title="Check out this tutorial"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="jsx-1925298512 jsx-2352943703 tile-content-container"
        >
          <h3
            className="jsx-1925298512 jsx-2352943703 title"
            style={
              Object {
                "marginBottom": "20px",
              }
            }
          >
            Check out this tutorial
          </h3>
          <div
            className="jsx-1925298512 jsx-2352943703 __html-blob-content-container__"
            dangerouslySetInnerHTML={
              Object {
                "__html": "<h1>Hello world...</h1>",
              }
            }
          />
        </div>,
        <JSXStyle
          css={
            Array [
              "div.jsx-1925298512,h3.jsx-1925298512,ul.jsx-1925298512,li.jsx-1925298512{
  margin: 0;
  padding: 0;
;}",
              ".root.jsx-1925298512{box-shadow: 0px 0px 5px 0px rgba(88, 88, 88, 0.5); background:white;}",
              ".tile-content-container.jsx-1925298512{padding:40px;}",
              ".title.jsx-1925298512{font-family:adobe-garamond-pro, serif;font-size:20px;color:#415671;border-bottom:1px solid #d9dee5;font-weight:300;padding-bottom:20px;}",
              "ul.jsx-1925298512{list-style-type:none;font-family:brandon-grotesque, sans-serif;color:#415671;text-transform:uppercase;font-size:10px;font-weight:bold;padding-top:20px;}",
              ".author-name.jsx-1925298512{display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-box-pack:justify;-webkit-justify-content:space-between;-ms-flex-pack:justify;justify-content:space-between;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;}",
              ".__html-blob-content-container__.jsx-1925298512,.read-duration.jsx-1925298512{display:none;}",
              "@media screen and (min-width: 768px){.__html-blob-content-container__.jsx-1925298512,.read-duration.jsx-1925298512{display:block;}.author-name.jsx-1925298512 img.jsx-1925298512,.author-name.jsx-1925298512 span.jsx-1925298512{display:none;}.tile-content-container.jsx-1925298512{padding:40px 50px;}ul.jsx-1925298512{display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;padding:10px 0;margin-bottom:20px;border-bottom:1px solid #d9dee5;}.read-duration.jsx-1925298512{border-right:1px solid #d9dee5;padding-right:10px;margin-right:10px;}}",
              "@media screen and (min-width: 1024px){.tile-content-container.jsx-1925298512{padding:80px;}.title.jsx-1925298512{font-size:24px;}}",
            ]
          }
          styleId="1925298512"
        />,
        <JSXStyle
          css={
            Array [
              ".__html-blob-content-container__.jsx-2352943703 *{
  margin: 0;
  padding: 0;
 font-family:adobe-garamond-pro, serif;color:#41566f;vertical-align:top;}",
              ".__html-blob-content-container__.jsx-2352943703 p{line-height:1.5;}",
              ".__html-blob-content-container__.jsx-2352943703 .contentImageEmbedDIV{text-align:center;}",
              ".__html-blob-content-container__.jsx-2352943703 .contentImageEmbedDIV .contentImageEmbed{padding:1px;border:1px solid #d9dee5;margin-bottom:10px;width:90%;}",
              ".__html-blob-content-container__.jsx-2352943703 .videoEmbedIFrame{margin-bottom:10px;width:90%;}",
              ".__html-blob-content-container__.jsx-2352943703 .contentImageEmbedCaption,.videoEmbedCaption{font-size:10px;}",
              ".__html-blob-content-container__.jsx-2352943703 .videoEmbedDIV{text-align:center;}",
            ]
          }
          styleId="2352943703"
        />,
      ],
      "className": "jsx-1925298512 jsx-2352943703 root",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h3
              className="jsx-1925298512 jsx-2352943703 title"
              style={
                Object {
                  "marginBottom": "20px",
                }
              }
            >
              Check out this tutorial
            </h3>,
            <div
              className="jsx-1925298512 jsx-2352943703 __html-blob-content-container__"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<h1>Hello world...</h1>",
                }
              }
            />,
          ],
          "className": "jsx-1925298512 jsx-2352943703 tile-content-container",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Check out this tutorial",
              "className": "jsx-1925298512 jsx-2352943703 title",
              "style": Object {
                "marginBottom": "20px",
              },
            },
            "ref": null,
            "rendered": "Check out this tutorial",
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "jsx-1925298512 jsx-2352943703 __html-blob-content-container__",
              "dangerouslySetInnerHTML": Object {
                "__html": "<h1>Hello world...</h1>",
              },
            },
            "ref": null,
            "rendered": null,
            "type": "div",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "css": Array [
            "div.jsx-1925298512,h3.jsx-1925298512,ul.jsx-1925298512,li.jsx-1925298512{
  margin: 0;
  padding: 0;
;}",
            ".root.jsx-1925298512{box-shadow: 0px 0px 5px 0px rgba(88, 88, 88, 0.5); background:white;}",
            ".tile-content-container.jsx-1925298512{padding:40px;}",
            ".title.jsx-1925298512{font-family:adobe-garamond-pro, serif;font-size:20px;color:#415671;border-bottom:1px solid #d9dee5;font-weight:300;padding-bottom:20px;}",
            "ul.jsx-1925298512{list-style-type:none;font-family:brandon-grotesque, sans-serif;color:#415671;text-transform:uppercase;font-size:10px;font-weight:bold;padding-top:20px;}",
            ".author-name.jsx-1925298512{display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-box-pack:justify;-webkit-justify-content:space-between;-ms-flex-pack:justify;justify-content:space-between;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;}",
            ".__html-blob-content-container__.jsx-1925298512,.read-duration.jsx-1925298512{display:none;}",
            "@media screen and (min-width: 768px){.__html-blob-content-container__.jsx-1925298512,.read-duration.jsx-1925298512{display:block;}.author-name.jsx-1925298512 img.jsx-1925298512,.author-name.jsx-1925298512 span.jsx-1925298512{display:none;}.tile-content-container.jsx-1925298512{padding:40px 50px;}ul.jsx-1925298512{display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;padding:10px 0;margin-bottom:20px;border-bottom:1px solid #d9dee5;}.read-duration.jsx-1925298512{border-right:1px solid #d9dee5;padding-right:10px;margin-right:10px;}}",
            "@media screen and (min-width: 1024px){.tile-content-container.jsx-1925298512{padding:80px;}.title.jsx-1925298512{font-size:24px;}}",
          ],
          "styleId": "1925298512",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "css": Array [
            ".__html-blob-content-container__.jsx-2352943703 *{
  margin: 0;
  padding: 0;
 font-family:adobe-garamond-pro, serif;color:#41566f;vertical-align:top;}",
            ".__html-blob-content-container__.jsx-2352943703 p{line-height:1.5;}",
            ".__html-blob-content-container__.jsx-2352943703 .contentImageEmbedDIV{text-align:center;}",
            ".__html-blob-content-container__.jsx-2352943703 .contentImageEmbedDIV .contentImageEmbed{padding:1px;border:1px solid #d9dee5;margin-bottom:10px;width:90%;}",
            ".__html-blob-content-container__.jsx-2352943703 .videoEmbedIFrame{margin-bottom:10px;width:90%;}",
            ".__html-blob-content-container__.jsx-2352943703 .contentImageEmbedCaption,.videoEmbedCaption{font-size:10px;}",
            ".__html-blob-content-container__.jsx-2352943703 .videoEmbedDIV{text-align:center;}",
          ],
          "styleId": "2352943703",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="jsx-1925298512 jsx-2352943703 tile-content-container"
          >
            <h3
              className="jsx-1925298512 jsx-2352943703 title"
              style={
                Object {
                  "marginBottom": "20px",
                }
              }
            >
              Check out this tutorial
            </h3>
            <div
              className="jsx-1925298512 jsx-2352943703 __html-blob-content-container__"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<h1>Hello world...</h1>",
                }
              }
            />
          </div>,
          <JSXStyle
            css={
              Array [
                "div.jsx-1925298512,h3.jsx-1925298512,ul.jsx-1925298512,li.jsx-1925298512{
  margin: 0;
  padding: 0;
;}",
                ".root.jsx-1925298512{box-shadow: 0px 0px 5px 0px rgba(88, 88, 88, 0.5); background:white;}",
                ".tile-content-container.jsx-1925298512{padding:40px;}",
                ".title.jsx-1925298512{font-family:adobe-garamond-pro, serif;font-size:20px;color:#415671;border-bottom:1px solid #d9dee5;font-weight:300;padding-bottom:20px;}",
                "ul.jsx-1925298512{list-style-type:none;font-family:brandon-grotesque, sans-serif;color:#415671;text-transform:uppercase;font-size:10px;font-weight:bold;padding-top:20px;}",
                ".author-name.jsx-1925298512{display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-box-pack:justify;-webkit-justify-content:space-between;-ms-flex-pack:justify;justify-content:space-between;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;}",
                ".__html-blob-content-container__.jsx-1925298512,.read-duration.jsx-1925298512{display:none;}",
                "@media screen and (min-width: 768px){.__html-blob-content-container__.jsx-1925298512,.read-duration.jsx-1925298512{display:block;}.author-name.jsx-1925298512 img.jsx-1925298512,.author-name.jsx-1925298512 span.jsx-1925298512{display:none;}.tile-content-container.jsx-1925298512{padding:40px 50px;}ul.jsx-1925298512{display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;padding:10px 0;margin-bottom:20px;border-bottom:1px solid #d9dee5;}.read-duration.jsx-1925298512{border-right:1px solid #d9dee5;padding-right:10px;margin-right:10px;}}",
                "@media screen and (min-width: 1024px){.tile-content-container.jsx-1925298512{padding:80px;}.title.jsx-1925298512{font-size:24px;}}",
              ]
            }
            styleId="1925298512"
          />,
          <JSXStyle
            css={
              Array [
                ".__html-blob-content-container__.jsx-2352943703 *{
  margin: 0;
  padding: 0;
 font-family:adobe-garamond-pro, serif;color:#41566f;vertical-align:top;}",
                ".__html-blob-content-container__.jsx-2352943703 p{line-height:1.5;}",
                ".__html-blob-content-container__.jsx-2352943703 .contentImageEmbedDIV{text-align:center;}",
                ".__html-blob-content-container__.jsx-2352943703 .contentImageEmbedDIV .contentImageEmbed{padding:1px;border:1px solid #d9dee5;margin-bottom:10px;width:90%;}",
                ".__html-blob-content-container__.jsx-2352943703 .videoEmbedIFrame{margin-bottom:10px;width:90%;}",
                ".__html-blob-content-container__.jsx-2352943703 .contentImageEmbedCaption,.videoEmbedCaption{font-size:10px;}",
                ".__html-blob-content-container__.jsx-2352943703 .videoEmbedDIV{text-align:center;}",
              ]
            }
            styleId="2352943703"
          />,
        ],
        "className": "jsx-1925298512 jsx-2352943703 root",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h3
                className="jsx-1925298512 jsx-2352943703 title"
                style={
                  Object {
                    "marginBottom": "20px",
                  }
                }
              >
                Check out this tutorial
              </h3>,
              <div
                className="jsx-1925298512 jsx-2352943703 __html-blob-content-container__"
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "<h1>Hello world...</h1>",
                  }
                }
              />,
            ],
            "className": "jsx-1925298512 jsx-2352943703 tile-content-container",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Check out this tutorial",
                "className": "jsx-1925298512 jsx-2352943703 title",
                "style": Object {
                  "marginBottom": "20px",
                },
              },
              "ref": null,
              "rendered": "Check out this tutorial",
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "jsx-1925298512 jsx-2352943703 __html-blob-content-container__",
                "dangerouslySetInnerHTML": Object {
                  "__html": "<h1>Hello world...</h1>",
                },
              },
              "ref": null,
              "rendered": null,
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "css": Array [
              "div.jsx-1925298512,h3.jsx-1925298512,ul.jsx-1925298512,li.jsx-1925298512{
  margin: 0;
  padding: 0;
;}",
              ".root.jsx-1925298512{box-shadow: 0px 0px 5px 0px rgba(88, 88, 88, 0.5); background:white;}",
              ".tile-content-container.jsx-1925298512{padding:40px;}",
              ".title.jsx-1925298512{font-family:adobe-garamond-pro, serif;font-size:20px;color:#415671;border-bottom:1px solid #d9dee5;font-weight:300;padding-bottom:20px;}",
              "ul.jsx-1925298512{list-style-type:none;font-family:brandon-grotesque, sans-serif;color:#415671;text-transform:uppercase;font-size:10px;font-weight:bold;padding-top:20px;}",
              ".author-name.jsx-1925298512{display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-box-pack:justify;-webkit-justify-content:space-between;-ms-flex-pack:justify;justify-content:space-between;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;}",
              ".__html-blob-content-container__.jsx-1925298512,.read-duration.jsx-1925298512{display:none;}",
              "@media screen and (min-width: 768px){.__html-blob-content-container__.jsx-1925298512,.read-duration.jsx-1925298512{display:block;}.author-name.jsx-1925298512 img.jsx-1925298512,.author-name.jsx-1925298512 span.jsx-1925298512{display:none;}.tile-content-container.jsx-1925298512{padding:40px 50px;}ul.jsx-1925298512{display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;padding:10px 0;margin-bottom:20px;border-bottom:1px solid #d9dee5;}.read-duration.jsx-1925298512{border-right:1px solid #d9dee5;padding-right:10px;margin-right:10px;}}",
              "@media screen and (min-width: 1024px){.tile-content-container.jsx-1925298512{padding:80px;}.title.jsx-1925298512{font-size:24px;}}",
            ],
            "styleId": "1925298512",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "css": Array [
              ".__html-blob-content-container__.jsx-2352943703 *{
  margin: 0;
  padding: 0;
 font-family:adobe-garamond-pro, serif;color:#41566f;vertical-align:top;}",
              ".__html-blob-content-container__.jsx-2352943703 p{line-height:1.5;}",
              ".__html-blob-content-container__.jsx-2352943703 .contentImageEmbedDIV{text-align:center;}",
              ".__html-blob-content-container__.jsx-2352943703 .contentImageEmbedDIV .contentImageEmbed{padding:1px;border:1px solid #d9dee5;margin-bottom:10px;width:90%;}",
              ".__html-blob-content-container__.jsx-2352943703 .videoEmbedIFrame{margin-bottom:10px;width:90%;}",
              ".__html-blob-content-container__.jsx-2352943703 .contentImageEmbedCaption,.videoEmbedCaption{font-size:10px;}",
              ".__html-blob-content-container__.jsx-2352943703 .videoEmbedDIV{text-align:center;}",
            ],
            "styleId": "2352943703",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
